src/broker/Manager.cc

```diff
@@ -652,7 +667,7 @@ bool Manager::PublishEvent(string topic, std::string name, broker::vector args)
 		return true;
 
 	DBG_LOG(DBG_BROKER, "Publishing event: %s", RenderEvent(topic, name, args).c_str());
-	broker::zeek::Event ev(std::move(name), std::move(args));
+	broker::zeek::Event ev(std::move(name), ts, std::move(args));
```

Comment:
Similarly as for the other API changes: Is this breaking backward compat of `broker::zeek::Event`?

Probably better for zeek/broker#331